.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,82 @@
+name: Build Wheels
+
+# Only run on new tags starting with `v`
+# on:
+#   push:
+#     tags:
+#       - 'v*'
+on: [push]
+
+jobs:
+  build_wheels:
+    name: Build wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: set environment variables
+      uses: allenevans/set-env@v1.1.0
+      with:
+        CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-*"
```

Comment:
You can change what Python versions wheels are built for, see https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip